pyproject.toml

```diff
@@ -225,7 +225,11 @@ lines-between-types = 0
 [tool.ruff.lint.per-file-ignores]
 "__init__.py" = ["F401", "E402", "F403", "F405", "E501", "I001"]
 "errors.py" = ["D105"]
-"tests/**.py" = ["D"]
+"tests/**.py" = ["D", "W505"]
```

Comment:
why are we ignoring for the tests?

Reply:
- When we enabled the length checking, I got error from the test directory.
- I figured that checking docstring length check (`W505`) was not important for the tests directory.

Reply:
I think we should apply it everywhere. The main use case as I see it is to make it easier to read and review, so I think tests would still have that problem

Reply:
Fixed:
https://github.com/sdv-dev/SDV/pull/2095/commits/02b804a5f2a397e4ca7179a5dc58f56980a193d6